public/app/features/dashboard-scene/utils/interactions.ts

```diff
@@ -137,12 +139,22 @@ export const DashboardInteractions = {
   toolbarAddClick: () => {
     reportDashboardInteraction('toolbar_actions_clicked', { item: 'add' });
   },
+
+  setScenesContext: () => {
+    isScenesContextSet = true;
+
+    return () => {
+      isScenesContextSet = false;
+    };
+  },
 };
 
 const reportDashboardInteraction: typeof reportInteraction = (name, properties) => {
+  const meta = isScenesContextSet ? { scenesView: true } : undefined;
+
   if (properties) {
-    reportInteraction(`dashboards_${name}`, properties);
+    reportInteraction(`dashboards_${name}`, properties, meta);
```

Comment:
Why do we need the extra param?
```suggestion
    reportInteraction(`dashboards_${name}`, {...properties, ...meta});
```